### ISSUE TITLE
Installation guide

### ISSUE THREAD

**Issue by dharmatech** [naivesound/glitch-orig#3]
Consider adding a note mentioning how to get `glitch` running on a user's own server.

Here's the output that resulted from running `npm install` on Ubuntu 15.10:

```
~/glitch $ npm install
npm WARN package.json @ No repository field.
npm ERR! git clone git@github.com:github:naivesound/expr-js Cloning into bare repository '/home/dharmatech/.npm/_git-remotes/git-github-com-github-naivesound-expr-js-ddc75adf'...
npm ERR! git clone git@github.com:github:naivesound/expr-js Warning: Permanently added the RSA host key for IP address '192.30.252.120' to the list of known hosts.
npm ERR! git clone git@github.com:github:naivesound/expr-js fatal: remote error: 
npm ERR! git clone git@github.com:github:naivesound/expr-js    is not a valid repository name
npm ERR! git clone git@github.com:github:naivesound/expr-js   Email support@github.com for help
npm ERR! notarget No compatible version found: expr@'github:naivesound/expr-js#1.0.3'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.0.0"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! System Linux 4.2.0-16-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! cwd /home/dharmatech/glitch
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.4.21
npm ERR! code ETARGET
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/dharmatech/glitch/npm-debug.log
npm ERR! not ok code 0
```

**Comment by zserge** [comment]
Ubuntu comes with a pretty outdated Node. I'm using Ubuntu 14.04 and I had to install node 5.8.0 and npm 3.8.2 to use recent node libraries. If you have a look here - https://nodejs.org/en/ - you will see that Node.js nowadays is either v4 (stable) or v6 (under development).

Here are some instructions on how to install node and npm in debian/ubuntu - https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions

I have not been adding such instructions because I assumed that if a developer wants to deal with code - he already has node and npm installed.

If we're talking of regular users here - I'm about to release standalone Electron variant of Glitch so it could run as a desktop app on any modern OS including Ubuntu. Which means unless you want to hack the code - you don't need to set up Node.js.

**Comment by dharmatech** [comment]
Thanks zserge! Those instructions for updating Node worked well. I was able to `npm install` successfully.

Regarding users installing glitch, I was mainly interested in maybe messing around with adding my own functions.

**Comment by dharmatech** [comment]
Looks like this is being addressed in #8.
